src/script/standard.h

```diff
@@ -38,6 +39,17 @@ class CScriptID : public BaseHash<uint160>
  */
 static const unsigned int MAX_OP_RETURN_RELAY = 83;
 
+/**
+ * Allows up to one annex data to be embedded in a transaction, counting towards the
+ * same data carrier limit as OP_RETURN values
+ */
+extern bool accept_annex_data;
+
+/**
+ * Static maximum of annex data per transaction
+ */
+static const unsigned int MAX_ANNEX_DATA = 126;
```

Comment:
If it wasn't for this limitation, inscriptions could also use the annex and with that become more chain-space efficient because there's no commit/reveal scheme needed anymore?

Reply:
This limitation is specifically in place so I don't have to rely on a more complex encoding ala https://github.com/bitcoin-inquisition/bitcoin/pull/9 to start experimentation with useful size annexes. It's not a principled limitation with respect to DoS concerns or anything.

Reply:
For larger sizes it can be a bit annoying, since currently taproot checksigs don't cover *other inputs* annexes. Limiting total annex to 126 bytes means a griefing counter-party can only do so much griefing. 

I'm agnostic to what the specific format is; I'm looking to experiment with small sizes.